src/checkbox/index.tsx

```diff
@@ -11,6 +11,7 @@ export interface CheckboxProps {
   readonly value?: string,
   readonly checked?: boolean,
   readonly disabled?: boolean,
+  readonly labelBeforeInput?: boolean,
```

Comment:
I would prefer `labelPosition` with an enum left / right

Reply:
I'll do that

Reply:
Done